dom.bs

```diff
@@ -1026,24 +1049,40 @@ seen from the definition above, an <a>event listener</a> is a more broad concept
 </dl>
 
 <p>The
-<dfn method for=EventTarget><code>addEventListener(<var>type</var>, <var>callback</var>, <var>capture</var>)</code></dfn>
+<dfn method for=EventTarget><code>addEventListener(<var>type</var>, <var>callback</var>, <var>options</var>)</code></dfn>
 method, when invoked, must run these steps:
 
 <ol>
  <li><p>If <var>callback</var> is null, terminate these steps.
 
+ <li>If <var>options</var> is of type boolean, let <var>capture</var> be
+ <var>options</var> and let <var>passive</var> be false.  Otherwise let
+ <var>capture</var> and <var>passive</var> be the corresponding values in the
+ <var>options</var> {{EventListenerOptions}} dictionary.
```

Comment:
Ok, fair enough.  How's [this](https://github.com/RByers/dom/commit/3de3245316a84943b737a7db77a6d305a98b0ba8)?

Also my developer confusion argument is probably academic because any reasonable developer should assume that boolean options default to false anyway :-)